test/Api.Test/Controllers/AccountsControllerTests.cs

```diff
@@ -91,14 +91,14 @@ public async Task PostPrelogin_WhenUserExists_ShouldReturnUserKdfInfo()
     }
 
     [Fact]
-    public async Task PostPrelogin_WhenUserDoesNotExist_ShouldDefaultToSha256And100000Iterations()
+    public async Task PostPrelogin_WhenUserDoesNotExist_ShouldDefaultToSha256And600000Iterations()
```

Comment:
Should this have a better name? 
```suggestion
    public async Task PostPrelogin_WhenUserDoesNotExist_ShouldDefaultToSha256AndDefaultIterations()
```
or 
```suggestion
    public async Task PostPrelogin_WhenUserDoesNotExist_Should_Set_PBKDF2_Defaults()
```

Reply:
Updated in [`ba90b92` (#3304)](https://github.com/bitwarden/server/pull/3304/commits/ba90b92476401b268f31e6b696c03700700cea1d)